onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -60,15 +60,15 @@
 
 namespace onnxruntime::optimizer_utils {
 
-static void FilterTransformers(std::vector<std::unique_ptr<GraphTransformer>>& transformers,
-                               const std::unordered_set<std::string>& transformers_to_disable) {
+static void FilterTransformers(InlinedVector<std::unique_ptr<GraphTransformer>>& transformers,
+                               const InlinedHashSet<std::string>& transformers_to_disable) {
   if (transformers_to_disable.empty()) return;
 
   transformers.erase(
       std::remove_if(transformers.begin(), transformers.end(),
-                     [&](const std::unique_ptr<GraphTransformer>& transformer) {
+                     [&, transformers_to_disable_end = transformers_to_disable.end()](const std::unique_ptr<GraphTransformer>& transformer) {
                        return !transformer ||
-                              transformers_to_disable.find(transformer->Name()) != transformers_to_disable.end();
+                              transformers_to_disable.find(transformer->Name()) != transformers_to_disable_end;
```

Comment:
>transformers_to_disable [](http://example.com/codeflow?start=30&length=23)

FYI, noticed that flat_hash_set has a contains() method:
[https://github.com/abseil/abseil-cpp/blob/7f850b3167fb38e6b4a9ce1824e6fabd733b5d62/absl/container/flat_hash_set.h#L370-L374](https://github.com/abseil/abseil-cpp/blob/7f850b3167fb38e6b4a9ce1824e6fabd733b5d62/absl/container/flat_hash_set.h#L370-L374)

Reply:
Agreed to keep as is on C++17 level